src/app/search.service.ts

```diff
@@ -18,7 +18,7 @@ export class SearchService {
   changed: EventEmitter<boolean>; // Value indicates if an interactive UI change triggered this.
 
   nonDefaultsActive: boolean;
-  selectedSortLabel: string;
+  selectedSortLabel: 'Most raised' | 'Match funds remaining' | 'Relevance' | null;
```

Comment:
```suggestion
  selectedSortLabel: 'Most raised' | 'Match funds remaining' | 'Relevance';
```
Per https://github.com/thebiggive/donate-frontend/pull/1088#discussion_r1182685483